prep-html-relative-pathing/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div>Click here to go to the home page</div>
```

Comment:
Oops! Looks like you are missing an anchor element here...

Reply:
Is this correct?
![image](https://user-images.githubusercontent.com/76795279/104363443-6c31cb00-54d2-11eb-843f-68da080f477b.png)